README.md

```diff
@@ -50,20 +50,22 @@ Data (and functions related to that data) shared across multiple pages.
 ### `Page.`
 `Page.Writing.Rate.Main`, `Page.Writing.Rate.Update`, `Page.Writing.Rate.Model.Decoder`
 
-A page on the site, which has its own URL. These are not reusable, and implemented using a combination of types from `Data` and modules from `Nri`.
+A page on the site, which has its own URL. These are not reusable, and implemented using a combination of types from `Data` and modules from `Nri`. Comments for usage instructions aren't required, as code isn't intended to be reusable.
```

Comment:
🎉  Just this morning, I was wondering why sometimes I needed doc comments and sometimes I didn't!! 😻